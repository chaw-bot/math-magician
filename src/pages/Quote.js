import React from 'react';

const Quote = () => (
  <div className="quote-page">
    <h1>today&apos;s quote....</h1>
    <p>
      When the seas of life get rough, remember your Lifeguard walks on water.
      - Unknown
    </p>
  </div>
);

export default Quote;
