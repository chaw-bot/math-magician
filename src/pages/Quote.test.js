import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

it('Render Header', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});