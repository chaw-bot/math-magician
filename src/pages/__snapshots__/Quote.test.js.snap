// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header 1`] = `
<div
  className="quote-page"
>
  <h1>
    today's quote....
  </h1>
  <p>
    When the seas of life get rough, remember your Lifeguard walks on water. - Unknown
  </p>
</div>
`;
